src/Files.App/Files.App.csproj

```diff
@@ -26,27 +28,17 @@
         <TieredCompilation>true</TieredCompilation>
         <TieredPgo>true</TieredPgo>
     </PropertyGroup>
+
     <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
         <DefineConstants>TRACE;DEBUG;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
     </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <PropertyGroup Condition="'$(Configuration)' != 'Debug'">
         <DefineConstants>TRACE;RELEASE;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
         <Optimize>true</Optimize>
     </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)'=='Store'">
-      <DefineConstants>TRACE;RELEASE;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
-      <Optimize>true</Optimize>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)'=='Preview'">
-      <DefineConstants>TRACE;RELEASE;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
-      <Optimize>true</Optimize>
-    </PropertyGroup>
-    <PropertyGroup Condition="'$(Configuration)'=='Stable'">
-      <DefineConstants>TRACE;RELEASE;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
-      <Optimize>true</Optimize>
-    </PropertyGroup>
```

Comment:
Don't we need these?

Reply:
I inverted those if statements from each ones to ‘!= Debug’. Properties other than Debug one have same contexts.

Reply:
I certainly should do as well in the two C++ projects as you can see how messy those project files are; there are incredibly literally the same contexts.